init.lua

```diff
@@ -65,18 +65,18 @@ minetest.register_craft({
 })
 
 -- former submods
-if minetest.is_yes(minetest.setting_get("digilines_enable_inventory") or true) then
+if minetest.settings:get_bool("digilines_enable_inventory", true) then
```

Comment:
That's not 0.4.x compatible.

Reply:
Giving an extra parameter won't cause a crash, what about adding a `~=false` and commenting it to remove it later.

Reply:
done